api/reset.js

```diff
@@ -1,6 +1,6 @@
 var fileDB = require('../db/file-queries')
 
-const startSources = require('./sources').startingLinksSources
+const startSources = require('../tutorials/sources').startingLinksSources
```

Comment:
Instead of using this, you should get it from DB.

Reply:
This file was automatically changed when I moved sources.js file to a different directory. Actually, we do not need a reset API because it was only used to automatically populate LinksFile table for a newly created user, so I think it can be deleted.

Reply:
In that case you can remove it, as long as we are populating the tutorial files properly.